tests/tests/browser/browser.ini

```diff
@@ -5,7 +5,7 @@
 [DEFAULT]
 support-files =
   head.js
-  ../urlbar_interventions-1.0.0.zip
+  ../urlbar_interventions-1.0a1.zip
```

Comment:
out of curiosity, why a1 instead of 1.0.1?

Reply:
1.0 is the initial release imo. (Hopefully it's the only release, but maybe not...) Given that, there are a few ways I can think of to set the version before release. There's no wrong way, so I'd be fine with any of these, but I like alpha best.

* A number < 1, like 0.9. I guess we would start off with 1 as the minor version and increment it each time we make a new build for QA.
* Beta, like 1.0b1. I like this because it's clear we're working toward a 1.0 release. But I don't think "beta" is the right term since we wouldn't want actual users using this or beta-testing it yet.
* Alpha, like 1.0a2. Similar to beta, it's clear we're working toward a 1.0 release. It's also clear it's pre-beta and still undergoing heavy testing.

Reply:
Oh, and about going from 1.0.0 to 1.0a1 (which is probably what you were actually asking about!) -- I should have started off at 1.0a1. My example repo starts off at 1.0.0 and that's how I kept it all the way through development of top sites. From now on I'd like to start at 1.0a1. mython brought this to my attention recently.